### PR TITLE
Homework1 task3 max and min values

### PR DESCRIPTION
**Pull request by IuliiaLanchuk**

### CODE REVIEW

File: homework1/task03/max_min/task03.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import sys
+from typing import Tuple
+
+
+def find_maximum_and_minimum(file_name: str) -> Tuple[int, int]:
+    """
+    I'm find min and max values in file.
+
+    :param file_name: input file name
+    :return: tuple of two values
```
Comment (Cjkjvfnby) [inline homework1/task03/max_min/task03.py]: In homework, it is ok to skip this type of docs. 

In real life `tuple of two values` duplicates the annotation and does not bring additional knowledge to the code.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]
Please deal with docstrings.

File: homework1/task03/max_min/task03.py (hunk 1 above)
Comment (IuliiaLanchuk) [reply homework1/task03/max_min/task03.py]: okey, thanks.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]
Please remove docstring from the `__init__`

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**